crates/bevy_ecs/src/schedule_v3/config.rs

```diff
@@ -87,9 +86,13 @@ pub trait IntoSystemSetConfig: sealed::IntoSystemSetConfig {
     #[doc(hidden)]
     fn into_config(self) -> SystemSetConfig;
     /// Add to the provided `set`.
+    #[track_caller]
     fn in_set(self, set: impl SystemSet) -> SystemSetConfig;
-    /// Don't add this set to the schedules's default set.
-    fn no_default_set(self) -> SystemSetConfig;
+    /// Add to the provided "base" `set`.
+    #[track_caller]
+    fn in_base_set(self, set: impl SystemSet) -> SystemSetConfig;
+    /// Don't add this set to the schedules's default base set.
```

Comment:
The doc strings and function name really don't match here.

Reply:
Oh I think this method is supposed to have "no" as its prefix, not "in".

Reply:
This one is supped to have `in` as the prefix. (as sets do not get added to the default base set by default). Adding to the default base set is an opt-in operation for sets. It was the doc comment that needed to change.